development/ts-migration-dashboard/app/components/App.tsx

```diff
@@ -25,8 +27,10 @@ const modulesById = allModules.reduce<Record<string, ModulePartitionChild>>(
   {},
 );
 const overallTotal = {
-  numConvertedModules: allModules.filter((module) => module.hasBeenConverted)
-    .length,
+  numModulesInTypeScript: allModules.filter(
+    (module) =>
+      module.hasBeenConvertedToTypeScript || module.wasOriginallyInTypeScript,
```

Comment:
This is what allows the progress bar to show _all_ files in TypeScript, not just ones that were converted.